### ISSUE TITLE
App is unable to download files properly with screen off

### ISSUE THREAD

**Issue by pilsnerbeer** [nextcloud/android#6230]
### Steps to reproduce
1. Download a reasonably big file that will take a few minutes to download (mine was 500MB)
2. Turn the screen off and wait for the file to download
3. Notice your files weren't downloaded and an error was printed instead

note: I dont use any specialized tools or settings that would kill NC when device switches to sleep. Battery optimization is set to default on Android 10 (adaptive).

### Expected behaviour
App should download files on the background or with screen off with no problems.

### Actual behaviour
![Screenshot_20200605-232622_1](https://user-images.githubusercontent.com/36133540/83940552-5e51f400-a7e5-11ea-927d-ebb1728477c2.jpg)

### Environment data
Android version: 10

Device model:  op5t

Nextcloud app version: 3.11.1

Nextcloud server version: 18.0.4

**Comment by tobiasKaminsky** [comment]
Android, or I guess in your case, Oxygen OS, seems to aggressively kill background processes.

This will be fixed by #6048

**Comment by ezaquarii** [comment]
@tobiasKaminsky I don't think this PR uses the downloader for anything other than contacts as of today
